src/test/java/org/broadinstitute/hellbender/tools/exome/ExomeToolsTestUtils.java

```diff
@@ -34,42 +34,57 @@ protected static File getTestDataDir(){
      */
     protected final static SAMSequenceDictionary REFERENCE_DICTIONARY = SAMSequenceDictionaryExtractor.extractDictionary(REFERENCE_FILE);
 
+    protected static Comparator<SimpleInterval> createIntervalComparator() {
```

Comment:
Yeah, that's what I was suggesting.  I don't know what a better name would be, although that one is a monstrosity.